surveys/models.py

```diff
@@ -36,6 +40,48 @@ class SurveyFormEntry(FormEntry):
     )
 
 
+class CensusObservation(models.Model):
+    fips_code = models.CharField(max_length=255)
+    variable = models.CharField(max_length=255)
+    fields = JSONField()
+
+
+class CensusArea(models.Model):
+    name = models.CharField(max_length=255)
+    fips_codes = pg_fields.ArrayField(models.CharField(max_length=12))
+
+    def __str__(self):
+        return self.name
+
+    def get_observations_from_component(self, component_name):
+        """
+        Given the name of a SurveyComponent, return the ACS observation data for this area.
+
+        :param component_name: The name of a SurveyComponent
+        :returns: A dictionary representing the stored data of the CensusObservation
+                  objects for the given area and SurveyComponent
+        """
```

Comment:
At first I tried directly associating `CensusObservation`s to `CensusArea`s by having a `ManyToManyField` called `CensusArea.observations`. This created a problem because a `CensusArea` is in a sense more abstract than a `CensusObservation`; it's a collection of FIPS codes, not a collection of specific observations.

This method is intended to help bridge the gap by returning `CensusObservations` for a given `CensusArea` based on the `CensusArea.fips_code` attribute and `SurveyComponent.name` ID. The use of `SurveyComponent.name` is a bit tricky, but makes sense for our use-case: The clearest way to define this relationship would be to use an ACS variable name instead (like `age_and_sex` or `household_tenure`), but since the `SurveyComponent.name` attribute is available to us directly from the survey submitted data object, it's a lot easier to use it to find `CensusObservation`s here.